cli/js/console_test.ts

```diff
@@ -306,6 +306,7 @@ test(function consoleTestCallToStringOnLabel(): void {
   for (const method of methods) {
     let hasCalled = false;
 
+    // @ts-ignore
```

Comment:
There should be a way to type it so we don't need the ignore.

Reply:
Similarly to https://github.com/denoland/deno/pull/3899#discussion_r377604819, the time of the argument passed to `console[method]` still wouldn't be correct so I'm not sure how to improve on this.

Reply:
If I type `methods` as `Array<keyof typeof Console>` (apparently `Console` refers to a value in this context) I then get the following error:
```
error TS7053: Element implicitly has an 'any' type because expression of type 'string | number | symbol' can't be used to index type 'Console'.
  No index signature with a parameter of type 'string' was found on type 'Console'.

► file:///Users/max/Developer/deno/cli/js/console_test.ts:310:5

310     console[method]({
        ~~~~~~~~~~~~~~~
```
And even if we got over that error, then we'd have the issue that `{ toString(): void { hasCalled = true; } }` wouldn't be considered a valid argument to these functions anyway (for instance, `count` only accepts a string), so I think the best thing to do is switch `methods` back to implicitly be of type `string[]` as before my changes and keep the `@ts-ignore`.

Reply:
🤷‍♂ ok! you win! :-D